docs/source/install/deb.rst

```diff
@@ -30,13 +30,12 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 Setup repositories
 ~~~~~~~~~~~~~~~~~~~
 
-  .. code-block:: bash
+The following script will detect your platform and architecture and setup the repo accordingly. It'll also install the GPG key for repo signing.
```

Comment:
@armab While with this, can you edit the link to st2bootstrap-deb.sh script at the top of the file to point to https://raw.githubusercontent.com/lakshmi-kannan/st2-packages/fix_bootstrap_script_to_use_pkgcloud/scripts/st2bootstrap-deb.sh instead of the formatted version? I can't get the link from docs and wget it directly.